app/tests/v1/test_meetup.py

```diff
@@ -51,17 +42,10 @@ def test_fetch_one_meetup(self):
         self.assertEqual(response_msg["status"], 200)
         self.assertTrue(response_msg["data"])
```

Comment:
Assert the contents of `response_msg["data"]` as well

Reply:
I have now asserted the contents of the response data by fetching the second meetup created.